### PR TITLE
ZEPPELIN-783: fix Spark download on CI

### PR DESCRIPTION
**Pull request by bzz**
### What is this PR for?
Fix Spark download on CI

### What type of PR is it?
Hot Fix

### What is the Jira issue?
[ZEPPELIN-783](https://issues.apache.org/jira/browse/ZEPPELIN-783)

### How should this be tested?
CI must be green

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by bzz** [comment]
\cc @felixcheung for review

**Comment by bzz** [comment]
This is fixes 2 CI https://travis-ci.org/bzz/incubator-zeppelin and https://travis-ci.org/apache/incubator-zeppelin and is ready to be merged.

\cc @felixcheung @Leemoonsoo @jongyoul @prabhjyotsingh for a review

**Comment by bzz** [comment]
Merging if there is no more discussion

**Comment by lresende** [comment]
LGTM

File: testing/downloadSpark.sh [hunk 1]
```diff
@@ -40,10 +40,28 @@ fi
 
 set -xe
 
-TIMEOUT_SEC=590
+MAX_DOWNLOAD_TIME_SEC=590
```
Comment (jongyoul) [inline testing/downloadSpark.sh]: Trivial: This isn't used any longer.

**Comment by jongyoul** [comment]
LGTM

File: testing/downloadSpark.sh (hunk 1 above)
Comment (bzz) [inline testing/downloadSpark.sh]: Good catch, but I thought about keeping it for a while, reminding about great use of `timeout` that we have discarded for now. We possibly might what get back to it one day (i.e to reduce build time), once we get the build stability.

**Comment by bzz** [comment]
Thank you for reviews!

**Comment by felixcheung** [comment]
Looks good - sorry for the delay.
My only thought was that we should probably echo the exit code from wget if it is not zero in download_with_retry()

**Pull request closed without merging**